### ISSUE TITLE
Recompute Weights

### ISSUE THREAD

**Issue by cortner** [ACEsuit/ACEpotentials.jl#125]
#124  adds a hack (not exported, and formally not part of the API) to recompute weights without recomputing the entire LSQ system. But this should become a better designed functionality within ACEfit

**Comment by cortner** [comment]
This would be best adressed by ACEsuit/ACEfit.jl#57 I think

**Comment by cortner** [comment]
can be closed once https://github.com/ACEsuit/ACEfit.jl/pull/58 is merged
